<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnConfig">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>160</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Config</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2"/>
   </widget>
   <widget class="QLabel" name="lblBus">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Bus</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbBus">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="prefix">
     <string>0x</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="displayIntegerBase">
     <number>16</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUnconfig">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>110</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Unconfig</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRefresh">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>240</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
